### ISSUE TITLE
How to reproduce the results on the partial point clouds?

### ISSUE THREAD

**Issue by hxwork** [pranavkdm/R-PointHop#6]
Hi, 

Thank you for releasing your codes, and it is a really excellent job! I wonder to reproduce the results on the partial-to-partial registration data, but I fail to find the settings of this experiment, such as the numbers of sample points and the numbers of the neighbors. Besides, I have no idea about that if the model needs to be re-trained on the partial data, or the released model can be used to register the partial inputs directly. Hope you can answer my doubts.

**Comment by pranavkdm** [comment]
Hi Hao,

Thanks for the compliment. The same model trained on complete point cloud objects was used for evaluating on partial point clouds. The procedure to generate the partial source and target point cloud is mentioned in the paper. You select a point at random and get it's 768 neighbors. The nearest neighbors in each stage may be set to 32. But you can try some values in that range to see what works best.

**Comment by hxwork** [comment]
Thank you for your quick reply. I will try to adjust the hyperparameters to see that if better results can be obtained. By the way, I have another question about the random seed during evaluation. Is the random seed fixed before the evaluation? Hope for your reply.

**Comment by pranavkdm** [comment]
No it's random. So you may expect some variations in results everytime.

**Comment by hxwork** [comment]
Thanks again, I will close this issue.
